code/game/machinery/doors/door.dm

```diff
@@ -323,9 +325,9 @@
 	operating = TRUE
 	do_animate("opening")
 	set_opacity(0)
-	sleep(0.5 SECONDS)
+	sleep(open_speed)
 	density = FALSE
-	sleep(0.5 SECONDS)
+	sleep(open_speed)
```

Comment:
timers

Reply:
a rewrite of that size is probably out of scope

Reply:
if open speed is only taken into consideration in this two procs i don't see why it would be out of scope to just move everything into their seperate proc?

Reply:
When someones only changing a number, asking them to rewrite the procs using that number is certainly out of scope in my opinion.

Reply:
"scope" this, "scope" that
take it as a learning experience and figure it out